domains/certificates/Shared/Contracts/Certificates/CertificateIssuedInProjectOriginEvent.cs

```diff
@@ -0,0 +1,6 @@
+using System;
+
+namespace Contracts.Certificates
+{
+    public record CertificateIssuedInProjectOriginEvent(Guid CertificateId);
```

Comment:
Please rename the event to be in past tense.

Reply:
It is? Issued